pkg/promclient/promclient.go

```diff
@@ -33,6 +33,7 @@ import (
 	"google.golang.org/grpc/codes"
 	"gopkg.in/yaml.v2"
 
+	"github.com/thanos-io/thanos/internal/cortex/querier/queryrange"
```

Comment:
Seems like this introduced an issue for Cortex when updating new Thanos version.
https://github.com/cortexproject/cortex/pull/5401 The `internal` package will be included as part of the Cortex vendor and I got some metrics registration conflict https://github.com/thanos-io/thanos/pull/6440.

I am wondering what would be a better way to go in the future. But for now, are we able to not include the `internal` package?

Reply:
https://github.com/thanos-io/thanos/pull/6440 Try to remove those duplicate metrics to make Cortex compilable